src/app/auth/forgot.tsx

```diff
@@ -0,0 +1,127 @@
+import { router } from 'expo-router';
+import React, { useState } from 'react';
+import { Alert, TextInput, View, StyleSheet } from 'react-native';
+import { Button, Input } from 'react-native-elements';
+
+import globalStyles from '../../styles/globalStyles';
+import { useSession } from '../../utils/AuthContext';
+
+function VerificationScreen() {
```

Comment:
Please rename this screen to be forgotPassword.tsx and update the function name accordingly (PascalCase for the function name)